src/index.ts

```diff
@@ -81,17 +81,24 @@ export const dupeReport = async ({
   // Diff the reports
 
   let dupeDiff = "";
+  let headerDiff = false;
   let hasDiff = false;
   let change = 0;
 
   diffLines(masterReport.trim(), localReport.trim(), {
     newlineIsToken: true
-  }).forEach(line => {
+  }).forEach((line, lineNum) => {
     let lineStart = " ";
     if (line.added) {
+      if (lineNum <= 4) {
```

Comment:
This is just based off the line length of the header... kinda fragile. I'll update it to be more robust later.